query/math.go

```diff
@@ -71,16 +80,16 @@ func (MathValuer) callTrigFunction(fn func(x float64) float64, arg0 interface{})
 }
 
 func (MathValuer) callRoundFunction(fn func(x float64) float64, arg0 interface{}) (interface{}, bool) {
-	var value float64
+
```

Comment:
I see you updated this with my advice, but I think that just manually inlining this would be better since the code itself is relatively small. That way we would be able to remove this entire function.

Reply:
I had new commit with the inlining the callRoundFunction as your advice, But I would like to ask it is better inline such like function or write it in separate functions?

Reply:
I think in this case it's better to inline it. The previous system had more complicated logic where you had to convert and handle different inputs types. Since it's now as simple as a single type cast, inlining it just means there's less indirection. If we find it too repetitive, we can always refactor it into a function later.